tests/unit/ferc_archiver_test.py

```diff
@@ -96,8 +84,4 @@ async def test_valid_years(module_name, test_spec, mocker):
     resources = [res async for res in archiver.get_resources()]
     # don't await these, just check to make sure they have right intention
     dbfs = [r for r in resources if r.__name__ == "get_year_dbf"]
-    xbrls = [r for r in resources if r.__name__ == "get_year_xbrl"]
```

Comment:
Should we replace these checks with ones for `archive_xbrl_for_form`?

Reply:
Probably!